src/sentry/api/endpoints/source_map_debug_blue_thunder_edition.py

```diff
@@ -643,6 +642,7 @@ def get_sdk_debug_id_support(event_data):
             "sentry.javascript.react",
             "sentry.javascript.react-native",
             "sentry.javascript.remix",
+            "sentry.javascript.solid",
```

Comment:
Maybe a silly question but why's this part of the PR?

Reply:
Not a silly question. Just a little improvement while we're in this file.